pkg/kubectl/cmd/cmd.go

```diff
@@ -505,6 +507,7 @@ func NewKubectlCommand(in io.Reader, out, err io.Writer) *cobra.Command {
 				replace.NewCmdReplace(f, ioStreams),
 				wait.NewCmdWait(f, ioStreams),
 				convert.NewCmdConvert(f, ioStreams),
+				templates.NormalizeAll(commands.NewDefaultCommand(k8sdeps.NewFactory())),
```

Comment:
I'm not a fan of `kubectl kustomize *`.  It's long, it feels too "cute", and the word "kustomize" is inconsistent with the command structure under kubectl.

I'd like to see the KEP updated with more details to describe the user flow of this under `kubectl *`.  I don't consider "we'll add this to kubectl" as demonstrating sufficient focus on experience, and having a well reasoned sub command structure should be justified before we merge this.

Reply:
The KEP might not be the correct forum for that discussion. The process of discussion and feedback has already occurred, and the KEP has been accepted and merged. Importantly, the current state of the KEP is "implemented", which is described in the KEP process as -- `implemented: The KEP has been implemented and is no longer actively changed.` I propose we use the feature/enhancement or this PR to address concerns.

Reply:
@smarterclayton We're going to write a separate doc with details for user flow of `kubectl kustomize` and address some other concerns as well.

Reply:
I disagree, since the KEP stated "to maybe add to kubectl but with no details about how".  KEP are living documents, and so since we hand waved over that there, the KEP at least deserves to be updated.  I don't see a reason *not* to have the discussion in the KEP, since the outcome of the discussion is going to be to ask you to update the KEP :)

Reply:
I second Clayton here, KEP was developed for an external tool and the naming there was ok. Including this into kubectl does require separate KEP and the name **must** match intention. If you look through all of the commands we currently have in `kubectl` every one of them clearly state intention of the user, which is a proper English word. `kustomize` breaks that rule and we cannot allow that to happen. At least `customize` could be a starting point but I'm not sure if that clearly states the intention.

Reply:
If I'm reading this correctly this additionally creates super long commands such as `kubectl kustomize edit add resource`, is that correct? I'm worried this will confuse users, b/c I'm seeing 3 separate actions being called out in a single command call.